test/src/test/java/jenkins/security/stapler/GetterMethodFilterTest.java

```diff
@@ -266,19 +265,19 @@ public void testWithReturnMultiple_map() throws Exception {
         assertNotReachable("testWithReturnMultiple/map/a/");
         assertNotReachable("testWithReturnMultiple/map/b/");
     }
-
-    @TestExtension
-    public static class TestWithReturnCoreObject extends AbstractUnprotectedRootAction {
-        public View.People getPeople() {
-            // provide an index jelly view
-            return new View.People(Jenkins.get());
-        }
-    }
-
-    @Test
-    public void testWithReturnCoreObject_people() throws Exception {
-        assertReachableWithoutOk("testWithReturnCoreObject/people/");
-    }
+//
+//    @TestExtension
+//    public static class TestWithReturnCoreObject extends AbstractUnprotectedRootAction {
+//        public View.People getPeople() {
+//            // provide an index jelly view
+//            return new View.People(Jenkins.get());
+//        }
+//    }
+//
+//    @Test
+//    public void testWithReturnCoreObject_people() throws Exception {
+//        assertReachableWithoutOk("testWithReturnCoreObject/people/");
+//    }
```

Comment:
just
```suggestion
```

Reply:
Just a reminder to look for an alternative to retain test coverage, see the section "Work in progress" in the PR comment.